frontend/lib/pages/tests/data-driven-onboarding.test.tsx

```diff
@@ -37,7 +37,7 @@ describe('Data driven onboarding', () => {
 
   it('shows suggestions when they exist', async () => {
     const pal = await simulateResponse({unitCount: 5});
-    pal.rr.getByText(/5 units/i);
+    pal.rr.getByText(/No registration found./i);
```

Comment:
Since we no longer show unit counts for unregistered buildings, I updated this front-end test a bit. Lmk if you think we need some more robust tests here! Could be good practice for me to add.

Reply:
Oh that's a good idea!  Feel free to do that, I think it _should_ be fairly straightforward but let me know if you run into roadblocks or anything.

Also, can you change the name of this test (the first argument to the `it()` function) to something more appropriate?  I'm not sure if this test is actually testing anything about suggestions...

Reply:
Er the part about the bad test name is totally my bad, I'm not sure why I called this test what I did.  Feel free to ignore my idea, hehe.